### ISSUE TITLE
NVDA and OEClassic 

### ISSUE THREAD

**Issue by BEdenhofer** [nvaccess/nvda#8195]
I've been speaking with an indavidual by the name of Zvonko Tesic, from the company who develops the OEClassic email client, and the problem with OEClassic is when you open it up, the message list view and the folder treeview isn't accessible with screenreaders, because it uses what's called a virtual treeview. Could we add support for that so OEClassic will be accessible with NVDA and other screenreaders? ### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open Chrome
> 2. Browse to www.google.com
> 3. Type "Hello"
> 4. Notice an error sound when enter is pressed.

> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.

### System configuration:
NVDA version:
> Example: next-14027,c80e529f

NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result

**Comment by Brian1Gaff** [comment]
Do they say other screenreaders do work with this then, or is this an issue 
for all of them. I think if its just nvda, then it stands a better chance of 
being looked at in case other software might benefit as well.


I wonder why they went down this road in the first place, seems like 
reinventing the wheel.
 Brian

**Comment by BEdenhofer** [comment]
Brian? Obviously, this is an issue with NVDA, otherwise, I wouldn't have reported it.

**Comment by Brian1Gaff** [comment]
I only asked if it affected just nvda or other screenreaders, it was not 
clear from what you wrote.
 Brian

**Comment by BEdenhofer** [comment]
I totaly understand now.

**Comment by leonardder** [comment]
It looks like this is a Delphi application. The virtual treeview component that is used to show the folders and message list doesn't seem to be very accessible (i.e. it doesn't expose every single item in the tree view as a child of the tree view itself). The only way NVDA can access the content is by means of the display model, which is far from ideal.

I'm labelling this needs external fix, as the baseline accessibility of these controls is just insufficient. I guess it could be fixed in NVDA if we really need to, but there are alternatives for this client (including thunderbird) that are much more accessible. Therefore, I'm also labelling this p4.

**Comment by Adriani90** [comment]
@leonardder could this possibly be touched by your display model PR?

**Comment by Adriani90** [comment]
In OE Classic 4.0 or higher, NVDA 2023.2 Beta 3 works properly for the message list and for the folder treeview. How ever, there is of course a big difference to Outlook and thunderbird as regarding accessibility. There are certainly still many bugs in OE classic, but this needs to be solved with the app developers first. NVDA supports these tree views apparently.

I am closing this issue as works for me. In case you have specific issues with OE classic, open separate issues for each action and provide detailed steps to reproduce.
